place_and_route/build_defs.bzl

```diff
@@ -60,6 +60,7 @@ def _place_and_route_impl(ctx):
 
     return [
         DefaultInfo(files = depset(output_files)),
+        SynthesisInfo(top_module = ctx.attr.synthesized_rtl[SynthesisInfo].top_module),
```

Comment:
It would be helpful if `place_and_route` rule would return SynthesisInfo provider with the name of the `top_module` from the synthesized design. This name is required in GDS write script, see e.g. [here](https://github.com/hdl/bazel_rules_hdl/pull/169/files/1b70039d8ff2dc0e741ba9a020a606b99d9dc023#diff-9a0852b1664f229eada1c6cf7973ba4e8f17edd051dfbe1b4b560bc7c52c3787R135)

Reply:
/cc @QuantamHD

Reply:
Why not just return the ctx.attr.synthesized_rtl[SynthesisInfo] directly there's really no harm.

Reply:
Done